requirements/runtime.txt

```diff
@@ -1,4 +1,5 @@
 matplotlib
 numpy
+opencv-python>=3
```

Comment:
From mmcv experiments, they don't recommend adding opencv in requirements as opencv have 4 versions and there is a conflict between them

Reply:
Delete `opencv-python>=3` would raise error.